mainnet/Dockerfile

```diff
@@ -35,7 +35,7 @@ WORKDIR /code
 
 # docker build --build-arg CACHEBUST=$(date +%s) .
 ARG CACHEBUST=1
-RUN git clone --branch v2.3.3 --single-branch --depth 1 https://github.com/QuarkChain/pyquarkchain.git
```

Comment:
`--single-branch --depth 1` will break the client version polling introduced in #512